### DISCUSSION TITLE
Could not open WPF app because get exception Could not load file or assembly '..\CefSharp.Core.Runtime.dll'

### DISCUSSION THREAD

**Discussion by phuocminh6464** [cefsharp/CefSharp#4507] [General]
Hi @amaitland,

I have a Windows PC running on Windows 10 Pro 64bit and I can use my WPF app normally. But when running app on Kiosk machine (Windows 11 Pro 64bit) exception error occurs although I'm using the same .msi installer file for both machine. This is exception details in Event Viewer

```
.NET Version: 7.0.4
Description: The process was terminated due to an unhandled exception.
Exception Info: System.IO.FileNotFoundException: Could not load file or assembly 'C:\Program Files\KioskApp\CefSharp.Core.Runtime.dll'. The specified module could not be found.
File name: 'C:\Program Files\KioskApp\CefSharp.Core.Runtime.dll'
   at CefSharp.CefSettingsBase..ctor()
   at CefSharp.Wpf.CefSettings..ctor() in C:\projects\cefsharp\CefSharp.Wpf\CefSettings.cs:line 16
   at xxx.App..ctor() in xxx\App.xaml.cs:line 35
   at xxx.App.Main()
```
Then I'm check that "CefSharp.Core.Runtime.dll" file exist in folder 'C:\Program Files\KioskApp'. Also, I try to run .exe file in Release folder but it still not open. Do you know what is it?
CefSharp Version: CefSharp.Wpf.NETCore (109.1.110)
Architecture: x64
.Net version: Net 7
Application: WPF

**Comment by BuslikDrev** [comment]
Установите VC++ 2015-2022 x86 и x64:
https://learn.microsoft.com/ru-ru/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022

**Comment by amaitland** [comment]
https://github.com/cefsharp/CefSharp/blob/cefsharp/113/NuGet/PackageReference/Readme.txt#L24

> CefSharp Version: CefSharp.Wpf.NETCore (109.1.110)

`M113` is the current supported version. Try upgrading.

https://github.com/cefsharp/CefSharp.MinimalExample is provided for testing purposes.
